django/cantusdb_project/main_app/admin/source.py

```diff
@@ -28,7 +28,7 @@ class SourceAdmin(BaseModelAdmin):
         "id",
         "provenance_notes",
     )
-    readonly_fields = READ_ONLY + (
+    readonly_fields = ("title", "siglum") + READ_ONLY + (
         "number_of_chants",
         "number_of_melodies",
         "date_created",
```

Comment:
```suggestion
    readonly_fields = READ_ONLY + (
        "title", 
        "siglum",
        "number_of_chants",
        "number_of_melodies",
        "date_created",
```

Reply:
I did it this way so that title and siglum appear at the top of the read-only field block